ocs_ci/ocs/utils.py

```diff
@@ -887,7 +887,12 @@ def run_must_gather(log_dir_path, image, command=None):
     # https://github.com/red-hat-storage/ocs-ci/issues/3240
     mg_output = ""
     ocs_version = version.get_semantic_ocs_version_from_config()
-    timeout = 1500 if ocs_version >= version.VERSION_4_6 else 600
+    if ocs_version < version.VERSION_4_6:
```

Comment:
@am-agrawa thanks for sending this PR. 
could you please make this change

`timeout = 1800 if ocs_version >= version.VERSION_4_11 else 1500`

version below 4.6 is unsupported and we won't run any tests in those versions.